### PR TITLE
Fix typos; add Raspbian lite

### PR DESCRIPTION
**Pull request by elnjensen**
Fixed miscellaneous typos; explain how to use command-line unzip on Mac
OS; mention Raspbian lite.

### CODE REVIEW

File: docs/docs/openaps-guide/core/medtronic.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ optional arguments:
                         Read current history pages.
     read_glucose_data   Read pump glucose page
     read_history_data   Read pump history page
-    read_insulin_sensitivies
+    read_insulin_sensitivities
```
Comment (scottleibrand) [inline docs/docs/openaps-guide/core/medtronic.md]: You can't fix this one.  It's the part of the help that documents the previously misspelled command.

**Comment by scottleibrand** [comment]
One line note: otherwise looks good to merge.

**Comment by elnjensen** [comment]
Thanks for catching that!  Sorry - didn't realize the history there.

**Comment by scottleibrand** [comment]
Just change it back on your branch and the PR will be updated automatically.

**Pull request merged**